students/colephalen/lesson01/assignment/inventory_management/electricAppliancesClass.py

```diff
@@ -0,0 +1,22 @@
+# Electric appliances class
+from inventoryClass import inventory
+
+class electricAppliances(inventory):
+
+    def __init__(self, productCode, description, marketPrice, rentalPrice, brand, voltage):
+        inventory.__init__(self,productCode,description,marketPrice,rentalPrice) # Creates common instance variables from the parent class
```

Comment:
Since inventory is the parent, you can get away with:

super().__init__(self,productCode,description,marketPrice,rentalPrice)